### ISSUE TITLE
Juniper rollback ignore warnings requires PyEZ 2.6.3

### ISSUE THREAD

**Issue by wasabi222** [napalm-automation/napalm#1597]
### Description of Issue/Question
this is pretty straightforward. When I go do dev.load_replace_candidate(filename='mycfg') I get the following (shortened) traceback:

TypeError: rollback() got an unexpected keyword argument 'ignore_warning'

this is because of this line: https://github.com/napalm-automation/napalm/blob/develop/napalm/junos/junos.py#L387

```
        self.device.cu.rollback(rb_id=0, ignore_warning=self.ignore_warning)
```

This prevents the config from being committed as well as leaves the session locked, which requires logging into the router and running `request system logout user hackerman`

The fix is just to remove the `ingore_warning` kwarg. PR incoming.

**Comment by wasabi222** [comment]
full traceback:

```
In [11]: dev.load_replace_candidate(filename='dev.cfg')
------------------------------------------------------------------------
RPCError                               Traceback (most recent call last)
/usr/local/lib/python3.7/dist-packages/jnpr/junos/device.py in execute(self, rpc_cmd, ignore_warning, **kvargs)
    836                 ignore_warning=ignore_warning,
--> 837                 filter_xml=kvargs.get("filter_xml"),
    838             )

/usr/local/lib/python3.7/dist-packages/jnpr/junos/decorators.py in wrapper(self, *args, **kwargs)
    164                 #  an XML attribute. Raise it up for the caller to deal with.
--> 165                 raise ex
    166         return rsp

/usr/local/lib/python3.7/dist-packages/jnpr/junos/decorators.py in wrapper(self, *args, **kwargs)
    116         try:
--> 117             rsp = function(self, *args, **kwargs)
    118         except RPCError as ex:

/usr/local/lib/python3.7/dist-packages/jnpr/junos/device.py in _rpc_reply(self, rpc_cmd_e, filter_xml)
   1439         if NCCLIENT_FILTER_XML:
-> 1440             return self._conn.rpc(rpc_cmd_e, filter_xml)._NCElement__doc
   1441         else:

/usr/local/lib/python3.7/dist-packages/ncclient/manager.py in execute(self, cls, *args, **kwds)
    230                    raise_mode=self._raise_mode,
--> 231                    huge_tree=self._huge_tree).request(*args, **kwds)
    232

/usr/local/lib/python3.7/dist-packages/ncclient/operations/third_party/juniper/rpc.py in request(self, rpc, filter_xml)
     51         self._filter_xml = filter_xml
---> 52         return self._request(rpc)
     53

/usr/local/lib/python3.7/dist-packages/ncclient/operations/rpc.py in _request(self, op)
    359                         else:
--> 360                             raise self._reply.error
    361                 if self._device_handler.transform_reply():

RPCError: syntax error

During handling of the above exception, another exception occurred:

RpcError                               Traceback (most recent call last)
/usr/local/lib/python3.7/dist-packages/jnpr/junos/utils/config.py in try_load(rpc_contents, rpc_xattrs, ignore_warning)
    490                 got = self.rpc.load_config(
--> 491                     rpc_contents, ignore_warning=ignore_warning, **rpc_xattrs
    492                 )

/usr/local/lib/python3.7/dist-packages/jnpr/junos/rpcmeta.py in load_config(self, contents, ignore_warning, **options)
    287
--> 288         return self._junos.execute(rpc, ignore_warning=ignore_warning)
    289

/usr/local/lib/python3.7/dist-packages/jnpr/junos/decorators.py in wrapper(*args, **kwargs)
     75             try:
---> 76                 return function(*args, **kwargs)
     77             except Exception:

/usr/local/lib/python3.7/dist-packages/jnpr/junos/decorators.py in wrapper(*args, **kwargs)
     30             try:
---> 31                 return function(*args, **kwargs)
     32             except Exception:

/usr/local/lib/python3.7/dist-packages/jnpr/junos/device.py in execute(self, rpc_cmd, ignore_warning, **kvargs)
    853                 rsp = None
--> 854             raise EzErrors.RpcError(cmd=rpc_cmd_e, rsp=rsp, errs=ex)
    855         # Something unexpected happened - raise it up

RpcError: RpcError(severity: error, bad_element: 2:50:34, message: syntax error)

During handling of the above exception, another exception occurred:

ConfigLoadError                        Traceback (most recent call last)
/usr/local/lib/python3.7/dist-packages/napalm/junos/junos.py in _load_candidate(self, filename, config, overwrite)
    266                 overwrite=overwrite,
--> 267                 ignore_warning=self.ignore_warning,
    268             )

/usr/local/lib/python3.7/dist-packages/jnpr/junos/utils/config.py in load(self, *vargs, **kvargs)
    578         if rpc_contents is not None:
--> 579             return try_load(rpc_contents, rpc_xattrs, ignore_warning=ignore_warning)
    580         elif "url" in kvargs and rpc_contents is None:

/usr/local/lib/python3.7/dist-packages/jnpr/junos/utils/config.py in try_load(rpc_contents, rpc_xattrs, ignore_warning)
    495             except RpcError as err:
--> 496                 raise ConfigLoadError(cmd=err.cmd, rsp=err.rsp, errs=err.errs)
    497             # Something unexpected happened - raise it up

ConfigLoadError: ConfigLoadError(severity: error, bad_element: 2:50:34, message: error: syntax error)

During handling of the above exception, another exception occurred:

TypeError                              Traceback (most recent call last)
<ipython-input-11-375e97ba16e5> in <module>
----> 1 dar.load_replace_candidate(filename='dar1.cfg')

/usr/local/lib/python3.7/dist-packages/napalm/junos/junos.py in load_replace_candidate(self, filename, config)
    277         """Open the candidate config and merge."""
    278         self.config_replace = True
--> 279         self._load_candidate(filename, config, True)
    280
    281     def load_merge_candidate(self, filename=None, config=None):

/usr/local/lib/python3.7/dist-packages/napalm/junos/junos.py in _load_candidate(self, filename, config, overwrite)
    268             )
    269         except ConfigLoadError as e:
--> 270             self.discard_config()
    271             if self.config_replace:
    272                 raise ReplaceConfigException(e.errs)

/usr/local/lib/python3.7/dist-packages/napalm/junos/junos.py in discard_config(self)
    385     def discard_config(self):
    386         """Discard changes (rollback 0)."""
--> 387         self.device.cu.rollback(rb_id=0, ignore_warning=self.ignore_warning)
    388         if not self.lock_disable and not self.session_config_lock:
    389             self._unlock()

TypeError: rollback() got an unexpected keyword argument 'ignore_warning'
```

I forgot to mention I tested the proposed fix above and it works as expected.

**Comment by wasabi222** [comment]
PR #1598

**Comment by ktbyers** [comment]
Fix is to require Juniper PyEZ 2.6.3

**Comment by ktbyers** [comment]
Fixed here:

https://github.com/napalm-automation/napalm/pull/1608
